doc/sf/logaddexp.qbk

```diff
@@ -0,0 +1,286 @@
+logsumexp_performance<double>/512/real_time                5312 ns         5307 ns       135435
+logsumexp_performance<double>/1024/real_time               9647 ns         9695 ns        69300
+logsumexp_performance<double>/2048/real_time              18521 ns        18386 ns        36542
+logsumexp_performance<double>/4096/real_time              37065 ns        37082 ns        18540
+logsumexp_performance<double>/8192/real_time              75715 ns        74801 ns         9400
+logsumexp_performance<double>/16384/real_time            156009 ns       154599 ns         4447
+logsumexp_performance<double>/32768/real_time            292937 ns       295307 ns         2381
+logsumexp_performance<double>/65536/real_time            584194 ns       582539 ns         1207
+logsumexp_performance<double>/131072/real_time          1175421 ns      1174428 ns          612
+logsumexp_performance<double>/262144/real_time          2382460 ns      2375422 ns          296
+logsumexp_performance<double>/524288/real_time          4720312 ns      4783163 ns          147
+logsumexp_performance<double>/1048576/real_time         9570338 ns      9457237 ns           76
+logsumexp_performance<double>/real_time_BigO               9.10 N          9.04 N
+logsumexp_performance<double>/real_time_RMS                   1 %             1 %
+logsumexp_performance<long double>/64/real_time             599 ns          599 ns      1147319
+logsumexp_performance<long double>/128/real_time           1163 ns         1166 ns       602863
+logsumexp_performance<long double>/256/real_time           2270 ns         2287 ns       307495
+logsumexp_performance<long double>/512/real_time           4523 ns         4491 ns       153067
+logsumexp_performance<long double>/1024/real_time          8833 ns         8845 ns        77731
+logsumexp_performance<long double>/2048/real_time         17855 ns        17582 ns        38214
+logsumexp_performance<long double>/4096/real_time         35755 ns        35488 ns        19813
+logsumexp_performance<long double>/8192/real_time         86561 ns        86875 ns         8813
+logsumexp_performance<long double>/16384/real_time       159225 ns       161993 ns         4244
+logsumexp_performance<long double>/32768/real_time       309235 ns       307948 ns         2334
+logsumexp_performance<long double>/65536/real_time       607855 ns       597465 ns         1203
+logsumexp_performance<long double>/131072/real_time     1164873 ns      1147023 ns          613
+logsumexp_performance<long double>/262144/real_time     2297293 ns      2297794 ns          306
+logsumexp_performance<long double>/524288/real_time     4664338 ns      4614094 ns          149
+logsumexp_performance<long double>/1048576/real_time    9226792 ns      9417808 ns           73
+logsumexp_performance<long double>/real_time_BigO          8.82 N          8.94 N
+logsumexp_performance<long double>/real_time_RMS              1 %             2 %
+```
+
+x86_64 Performance (Mingw GCC-11)
```

Comment:
Generally I only report one platform just to give people a general idea of how fast the function is-the reporting can be rather verbose and I think it may make people's eyes lose focus!

Also, these documents have been being built and read since 2006; we probably won't make any attempt to keep these performance numbers up to date and the CPUs will get better over time . . .